API.md

```diff
@@ -3394,6 +3394,18 @@ following options:
 - `noSniff` - boolean controlling the 'X-Content-Type-Options' header. Defaults to `true` setting
   the header to its only and default option, `'nosniff'`.
 
+- `referrer` - controls the ['Referrer-Policy'](https://www.w3.org/TR/referrer-policy/) header value:
+    - `false` - the 'Referrer-Policy' header will not be sent with responses. This is the default value.
+    - `''` - empty string indicating that the Referrer-Policy will be defined elsewhere.
```

Comment:
"elsewhere"?

Reply:
Also, can this be `"empty"` instead?

Reply:
Yes, we can change this, but it won't be consistent with the referrer policy options, which is to send a literal empty string. Up to you, do you want me to change this?

Reply:
It is just so odd to have an empty string as value. We can start with it as is and see if people are confused by it.

Reply:
"elsewhere" is the language used by w3, which really means in html meta-headers. I'll make this clearer